Changes

```diff
@@ -552,6 +552,10 @@ Working version
   cu_required_globals field of compilation unit descriptors.
   (Sébastien Hinderer, review by Vincent Laviron)
 
+- #????: Add Misc.print_see_manual and modify [@manual_ref] to accept
+  lists for simpler printing of manual references
+  (Stefan Muenzel, review by ????)
```

Comment:
This needs to be filled in before merging. (I almost merged.)

Reply:
Done! I left it purposefully vague in case someone other than @Octachron wanted to jump in to review